dotnet-algorand-sdk/V2/Indexer/SearchApi.cs

```diff
@@ -32,7 +32,7 @@ public partial interface ISearchApi
         /// <param name="application_id">Application ID</param>
         /// <returns>(empty)</returns>
         /// <exception cref="ApiException">A server side error occurred.</exception>
-        System.Threading.Tasks.Task<Response> AccountsAsync(int? asset_id, int? limit, string next, ulong? currency_greater_than, bool? include_all, ulong? currency_less_than, string auth_addr, ulong? round, int? application_id);
+        System.Threading.Tasks.Task<Response> AccountsAsync(ulong? asset_id, int? limit, string next, ulong? currency_greater_than, bool? include_all, ulong? currency_less_than, string auth_addr, ulong? round, int? application_id);
```

Comment:
if we're making assetid an ulong here, we should probably do the same on the actual asset model? (instead of bigint)

Reply:
we should =)